tests/lean/run/macro.lean

```diff
@@ -0,0 +1,70 @@
+import Init.Lean
+
+open Lean
+open Lean.Parser
+open Lean.Elab
+open Lean.Elab.Command
+
+@[commandParser] def syntax := parser! "syntax" >> many1 (termParser appPrec) >> " : " >> ident
+
+@[commandElab syntax] def elabSyntax : CommandElab :=
```

Comment:
These are mostly just minimum viable implementations for now. I haven't explored using `ParserDescr` and syntax cats yet.